### DISCUSSION TITLE
scale "lost guest" penalty treshold with guest count

### DISCUSSION THREAD

**Discussion by cheweytoo** [OpenRCT2/OpenRCT2#18645] [Feature requests]
With NSF, parks can now get pretty big, and guest counts can rise accordingly.

I propose to scale the "-7 per lost guest" penalty theshold by the total guest count: Leave it unchanged at 25 lost guests up to 7000 guests, but make it (number of guests/280) above. 280 because that's 7000/25, so it will neatly connect to 25 at a count of 7000 guests.

I have a park with a pretty decent path layout but a stable attendance of 13k guests, where the number of ~45-60 lost guests is insignificant, but annoyinlgy impacts the park rating.

**Comment by zalazarbiggins** [comment]
I'm running into this issue as well. Ideally guests wouldn't get lost at all if the paths are simple enough. I have a large park map with smaller pockets/zones that are connected with a grid. Even when the guest is traveling along a straight path that connects directly to their end destination they will get lost while walking said paths. I'm not sure what the game mechanics are, but it seems like a guest get lost after traveling a specific amount of spaces without reaching their desired destination. It might be nice to evolve that logic to also include number of turns taken. i.e. if the path has no turns, guests will not get lost no matter how many squares they traverse. If the guest must take X amount of turns in addition to a threshold squares traversed (keep it the same as the logic right now)—then they get lost.

IMO, ideally the penalty would would align to a % of the total guest count at a certain threshold. Perhaps their are tiers with different percentages to buffer the penalty as guest attendance increases so you'd always have some level of risk to mitigate, no matter the number of guests in the park. 

Not sure the answer, but it would be great to at least modify this penalty as a stop-gap solution. Ideally pathfinding is upgraded completely, but this fix would greatly improve game play. Like, I don't have to rely on cheats to lock the park rating because of lost guests—even when I've optimized path layout to avoid guests getting lost.
